xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-default/src/test/java/org/xwiki/refactoring/internal/DefaultModelBridgeTest.java

```diff
@@ -202,8 +215,36 @@ public void deleteTranslation() throws Exception
 
         this.modelBridge.delete(sourceReference);
 
-        verify(this.xcontext.getWiki()).deleteDocument(sourceDocument, this.xcontext);
-        assertLog(Level.INFO, "Document [{}] has been deleted.", sourceReference);
+        verify(this.xcontext.getWiki()).deleteDocument(sourceDocument, true, this.xcontext);
+        assertLog(Level.INFO, "Document [{}] has been deleted (trashed: [{}]).", sourceReference, true);
+    }
+
+    @Test
+    public void deleteAndSkipTheRecycleBin() throws Exception
```

Comment:
public keyword, not going to repeat it below

Reply:
Fixed (see https://github.com/xwiki/xwiki-platform/pull/1401/files#diff-29fdf5e5e3805db76947ad7e90902ac7R194)